### ISSUE TITLE
Been Trying for Hours and Still Getting Nowhere

### ISSUE THREAD

**Issue by blovelace** [MAVProxyUser/OriginalGangsterCow#3]
Hey there, I've been trying for hours on a Windows PC to get this installed onto the Crystal Sky for the Phantom 4 Pro+ (from these instructions https://www.youtube.com/watch?v=c-T61GKSRvk). I've enabled Developer options on the controller. I've enabled USB Debugging. I've enabled the developer mode on Windows 10 to enable Linux Bash Script Commands. I've downloaded every SDK made for Android along with the tools via Android Studio. I've pointed my PATH to the correct ADB Directory in Advanced System Settings. Every time I still get this error when trying to bash the script. "LastSkyCry.sh: line 3: adb: command not found"

![image](https://user-images.githubusercontent.com/29550663/33928829-1a88463e-df9c-11e7-9c36-9d4f1042b88c.png)

Is there something I'm doing wrong? 

Please advise! Thanks!

**Comment by MAVProxyUser** [comment]
Adb isn’t in your path I guess? Maybe try a real Linux instance?

**Comment by blovelace** [comment]
Yeah I ended up realizing that the Linux Shell in Windows can't communicate with USB devices. Pretty much the stupidest bug ever. Installed a VirtualBox running Debian and got everything working now!

**Comment by blovelace** [comment]
If anybody else runs into the same issue I did, feel free to follow my tutorial blog post here: http://brianlovelace.com/allow-3rd-party-apps-on-dji-phantom-4-pro-plus-tutorial/

**Comment by albfan** [comment]
Just in case this is helpful for anyone: There's a trick to install apks on DJI (adb install in the first second  after boot)

You can install flashfire that way and install google apps with zip from https://sourceforge.net/projects/opengapps/files/arm/20191003/open_gapps-arm-9.0-pico-20191003.zip/download

just that, will give you the google app store. But as nowadays `LastSkyCry.sh` leads to #5 there's not much you can do in that state.

@blovelace @MAVProxyUser any ideas?
